src/buffer.rs

```diff
@@ -939,6 +940,36 @@ where
     }
 }
 
+impl<P, Container> ImageBuffer<P, Container>
+where
+    P: Pixel + 'static,
+    [P::Subpixel]: EncodableLayout,
+    Container: Deref<Target = [P::Subpixel]>,
+{
+    /// Writes the buffer to a writer in the specified format.
+    ///
+    /// See [`ImageOutputFormat`](../enum.ImageOutputFormat.html) for
+    /// supported types.
+    ///
+    /// **Note**: TIFF encoding uses buffered writing,
+    /// which can lead to unexpected use of resources
+    pub fn write_to<W, F>(&self, writer: &mut W, format: F) -> ImageResult<()>
```

Comment:
Shouldn't this be called `save_to` to conform to the `save` method? If I understand correctly, `write` is only ever used internally, not in the public API